drivers/espi/Kconfig.xec

```diff
@@ -0,0 +1,19 @@
+# Kconfig.xec - Microchip CEC/MEC ESPI configuration options
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ESPI_XEC
+	bool "MEC/CEC Microchip ESPI driver"
+	depends on ESPI
+	help
+	Enable the Microchip MEC/MEC ESPI driver.
```

Comment:
Change the word MEC/CEC to XEC in all occurrences.  This is because the current CEC does not have an eSPI block